src/resonanceReconstruction/rmatrix/ChannelQuantumNumbers/test/ChannelQuantumNumbers.test.cpp

```diff
@@ -22,15 +22,14 @@ SCENARIO( "ChannelQuantumNumbers" ) {
     Parity pi = +1;
 
     THEN( "a ChannelQuantumNumbers can be constructed" ) {
+
       ChannelQuantumNumbers numbers( l, s, J, pi );
 
-      REQUIRE( 1 == numbers.orbitalAngularMomentum() );
-      REQUIRE( 0.5 == numbers.spin() );
-      REQUIRE( 1.5 == numbers.totalAngularMomentum() );
-      REQUIRE( +1 == numbers.parity() );
-      REQUIRE( "{1,1/2,3/2+}" == numbers.toString() );
-    }
+      CHECK( 1 == numbers.orbitalAngularMomentum() );
```

Comment:
You know this makes me happy

Reply:
Since I'm doing major refactoring, and since I have to go over all tests anyway, I thought I'd make that change at the same time.